addons/point_of_sale/static/src/js/Chrome.js

```diff
@@ -210,7 +210,7 @@ odoo.define('point_of_sale.Chrome', function(require) {
         __closeTempScreen() {
             this.tempScreen.isShown = false;
         }
-        __showScreen({ detail: { name, props = {} } }) {
+        __showScreen({ detail: { name, props = { SetScreenToOrder: true }} }) {
```

Comment:
You can remove this because in the line 225, you are first checking if the `SetScreenToOrder` is in the props.